consent/migrations/0005_auto_20210413_1644.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.19 on 2021-04-13 21:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('consent', '0004_datasharingconsenttextoverrides'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='datasharingconsent',
+            index=models.Index(fields=['username'], name='consent_dat_usernam_fae23a_idx'),
```

Comment:
What's behind this name for it?

Reply:
That's randomly generated by the makemigration tool.